camply/cli.py

```diff
@@ -128,34 +160,91 @@ def configure(context: CamplyContext, debug: bool) -> None:
 )
 
 
+@camply_command_line.command()
+@rec_area_argument
+@provider_argument
+@click.pass_obj
+def equipment_types(
+    context: CamplyContext,
+    rec_area: Optional[int] = None,
+    provider: str = DEFAULT_CAMPLY_PROVIDER,
+) -> None:
+    """
+    Retrieve a list of equipment supported by the current provider/recreaton area
+
+    Equipment are camping equipment that can be used at a campsite. Different providers
+    and recreation areas have different types of equipment for which reservations can be made.
+    """
```

Comment:
This is all looking very clean - it's a great implementation. I've got a couple questions on here but I'm close to getting this ready for merge. 

There's currently a GitHub Actions Secrets error that occurs when I merge directly to main from an outside repo - let's target a different branch on juftin/camply and then I'll go to main from there.

I'd like to get some usage of GoingToCamp documented - including how to use the new equipment-types command. I've got a placeholder here: https://github.com/juftin/camply/pull/156. If it's helpful we can merge this PR into that branch or merge both PRs into a new branch.

Reply:
Cool, let's make #156 the target for this branch to address both issues. And I'll take a look at #156 to see about adding more equipment-type documentation.